lib/address.ml

```diff
@@ -0,0 +1,27 @@
+open Bin_prot.Std
+
+type t = {
+  address : string;
+  port : int;
+}
+[@@deriving bin_io, eq]
+
+let address_of address =
+  match address with
```

Comment:
This match is unnecessary, can just do t.address or

```ocaml
let address_of { address ; _ } =
  address
```

Reply:
Yup, done this morning 😉